pcsx2/USB/usb-eyetoy/cam-windows.h

```diff
@@ -72,7 +72,7 @@ namespace usb_eyetoy
 
 		typedef void (*DShowVideoCaptureCallback)(unsigned char* data, int len, int bitsperpixel);
 
-		typedef struct
+		typedef struct MPEG_Buffer
```

Comment:
```suggestion
		struct
```
Afaik anonymous structs that are used as a variable type are OK, but the warning comes from `typedef`.

Reply:
That changes what this does

Define a `struct A` and typedef it to `b`
```cpp
typedef struct A {} b;
```

Define a variable b of type \<anonymous struct\>
```cpp
struct {} b;
```

The other way to fix it would be to use the fact that this is C++ and not C, and do `struct buffer_t {`